pkg/odo/cli/create/namespace/namespace.go

```diff
@@ -80,32 +82,51 @@ func (nco *NamespaceCreateOptions) Validate(ctx context.Context) error {
 
 // Run runs the namespace create command
 func (nco *NamespaceCreateOptions) Run(ctx context.Context) (err error) {
-	// Create the "spinner"
-	s := &log.Status{}
-
-	// If the --wait parameter has been passed, we add a spinner..
-	if nco.waitFlag {
-		s = log.Spinnerf("Waiting for %s to come up", nco.commandName)
-		defer s.End(false)
-	}
+	createSpinner := log.Spinnerf("Creating the %s %q", nco.commandName, nco.namespaceName)
```

Comment:
I guess you also need to update the doc automation tests as well:

```
  << Timeline                                                                                                                                                                                  
                                                                                                                                                                                               
  [FAILED] create_namespace.mdx                                                                                                                                                                
  Expected                                                                                                                                                                                     
      <string>:   (                                                                                                                                                                            
                """                                                                                                                                                                            
                ```console                                                                                                                                                                     
                $ odo create namespace odo-dev                                                                                                                                                 
      +         •  Creating the namespace "odo-dev"  ...
      +         ✓  Creating the namespace "odo-dev" [1s]
                ✓  Namespace "odo-dev" is ready for use
                ✓  New namespace created and now using namespace: odo-dev
                ... // 2 identical lines
                """
        )
       
  to be empty
  In [It] at: /home/asoro/work/projects/odo/tests/documentation/command-reference/doc_command_reference_createnamespace_test.go:42 @ 06/27/23 12:03:09.82
------------------------------
•

Summarizing 2 Failures:
  [FAIL] doc command reference odo create namespace To create a namespace resource [It] Creates a project resource for a kubernetes cluster
  /home/asoro/work/projects/odo/tests/documentation/command-reference/doc_command_reference_createnamespace_test.go:52
  [FAIL] doc command reference odo create namespace To create a namespace resource [It] Creates a namespace resource for a kubernetes cluster
  /home/asoro/work/projects/odo/tests/documentation/command-reference/doc_command_reference_createnamespace_test.go:42
```

Also, can you rebase your branch onto `main`?